Synview-backend/middleware/auth_middleware.ts

```diff
@@ -1,15 +1,12 @@
-import {create, verify} from "@zaubrik/djwt";
+import { create, verify } from "@zaubrik/djwt";
 import { generateKey, getToken } from "../utils/JWTHelpers.ts";
 import { UserPayloadSchema } from "../../common/schemas.ts";
-import {UserPayload} from "../../common/types.ts"
+import { UserPayload } from "../../common/types.ts";
 import { AppState } from "../../common/types.ts";
-import {Context} from "@oak/oak"
-
-let key: CryptoKey;
-
+import { Context } from "@oak/oak";
 
 export async function createToken(payload: any): Promise<string> {
-  key = await generateKey();
+  const key: CryptoKey = await generateKey();
```

Comment:
Maybe rename `generateKey` to `getKey` or similar? This naming suggests a new key is generate each call rather than being reused.